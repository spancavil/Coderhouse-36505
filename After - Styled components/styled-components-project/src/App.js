import ItemListContainer from './containers/ItemListContainer';
// import ItemDetailContainer from './containers/ItemDetailContainer';

function App() {
  return (
    <div className="App">
      <ItemListContainer greeting={"Hola"}/>
    </div>
  );
}

export default App;
