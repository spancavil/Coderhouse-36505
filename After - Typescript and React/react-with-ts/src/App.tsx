import { useState } from 'react';
import ItemCount from './components/ItemCount';
import Modal from './components/Modal';
import ItemListContainer from './containers/ItemListContainer';

function App() {
  const [modal, setModal] = useState<boolean>(true)

  const onAdd = (quantity: number) => {
    console.log("Productos agregados al cart: ", quantity)
  }

  return (
    <div className="App">

      <ItemCount stock = {13} onAdd = {onAdd} initial={3}/>
      {modal && <Modal onClose={setModal} style={{
        width: '200px',
        height: '400px',
        boxShadow: '10px 10px 17px -7px rgba(0,0,0,0.75)',
        color: 'black',
        backgroundColor: 'white',
      }} >
        <h2>Contenido del modal</h2>
      </Modal>
      }

      { <ItemListContainer message='Hola' count={10} /> }
    </div>
  );
}

export default App;
