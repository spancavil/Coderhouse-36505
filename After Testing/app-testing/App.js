import { NavigationContainer } from '@react-navigation/native';
import { createNativeStackNavigator } from '@react-navigation/native-stack';
import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
import ScreenA from './Screens/ScreenA';
import ScreenB from './Screens/ScreenB';

const Stack = createNativeStackNavigator();

export default function App() {

  return (
    <NavigationContainer>
      <Stack.Navigator initialRouteName='Screen A'>
        <Stack.Screen
          name='Screen A'
          component={ScreenA}
          title="Screen A"
        ></Stack.Screen>
        <Stack.Screen
          name='Screen B'
          component={ScreenB}
          title="Screen B"
        ></Stack.Screen>
      </Stack.Navigator>
    </NavigationContainer>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
