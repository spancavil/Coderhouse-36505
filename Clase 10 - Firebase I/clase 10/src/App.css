@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,700&display=swap');

* {
  margin: 0;
  top: 0;
  box-sizing: border-box;
  font-family: 'Lato';
}
