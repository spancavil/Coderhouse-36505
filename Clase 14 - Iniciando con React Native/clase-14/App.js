import { StatusBar } from 'expo-status-bar';
import { StyleSheet, View, Text} from 'react-native';

export default function App() {
  return (
    <View style={styles.container}>
      <Text style={styles.text}>Clase 14 - Introducci√≥n a React Native</Text>
      <StatusBar style="auto" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 0.5,
    backgroundColor: '#000',
    alignItems: 'center',
    justifyContent: 'center',
    width: '90%',
  },
  text: {
    color: '#fff',
    fontSize: 12,
  }
});
