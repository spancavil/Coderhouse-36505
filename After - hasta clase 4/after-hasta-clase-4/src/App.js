import './App.css';
import NavBar from './components/NavBar';
import ItemListContainer from './containers/ItemListContainer';

function App() {
  return (
    <>
      <NavBar/>
      <ItemListContainer greeting={"Bienvenidos a nuestra after app"}/>
    </>
    );
}

export default App;
