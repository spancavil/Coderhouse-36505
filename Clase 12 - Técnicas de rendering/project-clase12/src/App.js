import './App.css';
import Button from './components/Button';
import InLine from './components/InLine';
import InputConClase from './components/InputConClases';
import ReturnTemprano from './components/ReturnTemprano';
import SpreadingDePropiedades from './components/SpreadingDePropiedades';

function App() {
  return (
    <div className="App">
      <ReturnTemprano condition={false}/>
      <InLine condition={true}/>
      <Button color="red" letter='white'/>
      <InputConClase condition={true}/>
      <SpreadingDePropiedades condition={true}/>
    </div>
  );
}

export default App;
