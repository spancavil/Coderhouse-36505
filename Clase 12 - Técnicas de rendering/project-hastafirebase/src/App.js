import './App.css';
import NavBar from './components/NavBar';
import ItemDetailContainer from './containers/ItemDetailContainer';
import ItemListContainer from './containers/ItemListContainer';
import { BrowserRouter, Route, Routes} from 'react-router-dom';
import NotFound from './components/NotFound';
import Cart from './components/Cart';
// import TestContainer from './containers/TestContainer';

function App() {
  return (
    <BrowserRouter>
      <NavBar/>
      <Routes>
        <Route path = '/' element = {<ItemListContainer/>}/>
        <Route path = '/category/:id' element = {<ItemListContainer/>} />
        <Route path = '/item/:id' element = {<ItemDetailContainer/>} />
        <Route path = '/cart' element={<Cart/>} />
        <Route path = '*' element = {<NotFound/>} />
      {/* <ItemListContainer greeting={"Bienvenidos a nuestra after app"}/> */}
      {/* <ItemDetailContainer/> */}
      </Routes>
    </BrowserRouter>
    );
}

export default App;
