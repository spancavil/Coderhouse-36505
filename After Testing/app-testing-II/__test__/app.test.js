import React from 'react';
import renderer from 'react-test-renderer';

import App from '../App';

describe('Testing <App/>', ()=> {

    let tree

    beforeEach(()=> {
        tree = renderer.create(<App />).toJSON();
    })

    it('Renderiza segÃºn el snapshot', () => {
        expect(tree).toMatchSnapshot();
    });

    it('Tiene un child', ()=> {
        expect(tree.children.length).toBe(2);
    })

})

