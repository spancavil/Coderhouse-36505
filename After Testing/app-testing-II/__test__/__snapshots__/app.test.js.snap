// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <App/> Renderiza seg√∫n el snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Coderhouse - testeando con Jest
  </Text>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Boton
      </Text>
    </View>
  </View>
</View>
`;
